schemas/control-statement-schema.json

```diff
@@ -0,0 +1,291 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "A Control Statement is a statement by a company or entity about a the control of a company. In it's most straightforward, Company A may say that Person B is the beneficial owner of it. Each statement should be about a single control relationship, i.e. if Company A has two beneficial owners, Persons B and C, and those the beneficial ownerships relationships are distinct (e.g. they started at a different time) then they should each be stored as a indiviudal Beneficial Ownership statements. They should only be stored in the same BO statement if it is logically part of the same control mechanism, e.g. if company A is controlled by company B, and the majority of the shares in B are *jointly* held by person C and person D",
+  "properties": {
+    "subject_entity": {
+      "$ref": "includes/entity.json"
+    },
+    "control_level": {
+      "description": "The level of control, i.e. whether the control is direct (i.e. there are no intermediaries), indirect (there are intermediaries) or unknown. It can be null n the case where a company is saying that there are no beneficial owners",
+      "enum": [
+        "direct",
+        "indirect",
+        "unknown",
+        "null"
+      ]
+    },
+    "ultimate_beneficial_owner": {
```

Comment:
What about `ultimate_control`? Just thinking we don't want to mention "beneficial owner" because of the difficulties around defining it.